File: Feitos em Java/2164.java
import java.io.IOException;
import static java.lang.Math.pow;
import static java.lang.Math.sqrt;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        double n1=(1+sqrt(5))/2,n2=(1-sqrt(5))/2;
    	int n=entrada.nextInt();
    	double total= (pow(n1,n)-pow(n2,n))/sqrt(5);
        System.out.printf("%.1f\n",total);
    }
}

File: Feitos em C++/2164.cpp
#include<iostream>
#include <stdio.h>
#include <math.h>
using namespace std;
int main(){
    double n1=(1+sqrt(5))/2,n2=(1-sqrt(5))/2;
	int n;
	cin>>n;
	double total= (pow(n1,n)-pow(n2,n))/sqrt(5);
	printf("%.1f\n",total);
    return 0;
}
